src/smartcard.rs

```diff
@@ -54,6 +54,7 @@ impl SmartCard {
         })
     }
 
+    #[allow(dead_code)]
```

Comment:
question: Why this code is dead? Can you add a comment explaining why we should keep the code around? Thank you!

Reply:
this one is also related to [my comment above](https://github.com/Devolutions/sspi-rs/pull/260#discussion_r1672888320):

> ...removing the smartcard layer...

Reply:
Indeed! In general, we avoid keeping dead code around so if we decide to do so here, I would like a FIXME comment so we don’t forget forever about it. In this case, I believe you’ll end up re-using these functions later so it’s fine.

Reply:
I've added FIXME